### DISCUSSION TITLE
No media found error

### DISCUSSION THREAD

**Discussion by robc006** [TheAcharya/MarkersExtractor#105] [Q&A] [answered]
I love this tool! I have everything installed and setup correctly as far as I can tell. I followed the instructions on setting up the .sh file as well. When I execute the command I don't get any images created from my markers on the clips in my timeline that I exported my xml from. I get the error that says No media found for "xmlfile". The media is defiantly on the timeline and the markers are found in the xml as well. 

Is there something I did wrong. I would love it if I could get this work. Awesome work guys!!!

Rob

**Comment by orchetect** [comment, marked as answer]
The extractor doesn't read media used in the FCPXML file since it does not do any visual compositing -- that's FCP's job. (Aside from the optional metadata labels you choose for the thumbnails of course).

It relies on a single exported video of your timeline. Once exported, the filename should match the base filename of the FCPXML file and reside in the same folder as the FCPXML file. If you can't copy or move it there, you can supply the exported video's containing folder path to the `--media-search-path <path>` CLI flag.

**Comment by IAmVigneswaran** [comment]
@robc006 Marker is now live. You can check it out.

https://github.com/TheAcharya/MarkerData
